### PR TITLE
Add functionality to share to instagram

### PR DESCRIPTION
**Pull request by lucyxubroad**
IMG_C11C0C2FD1BE-1

### CODE REVIEW

**Review by haiyingweng — APPROVED** [review]
yayyy

File: Flick/Views/ModalViews/ConfirmationModalView.swift [hunk 1]
```diff
@@ -40,7 +40,7 @@ class ConfirmationModalView: ModalView {
         messageLabel.lineBreakMode = .byWordWrapping
         containerView.addSubview(messageLabel)
 
-        purpleButton = RoundedButton(style: .purple, title: type == .clearIdeas ? "Clear" : "No")
+        purpleButton = RoundedButton(style: .purple, title: type == .clearIdeas ? "Clear" : "Yes")
```
Comment (haiyingweng) [inline Flick/Views/ModalViews/ConfirmationModalView.swift:43]: do we want to change this?

Comment (haiyingweng) [reply Flick/Views/ModalViews/ConfirmationModalView.swift:43]: oh wait i think this is good

Comment (haiyingweng) [reply Flick/Views/ModalViews/ConfirmationModalView.swift:43]: actually this is good haha

**Comment by cindyh25** [comment]
omg this is so cool!! 🙌

**Pull request merged**